packages/table/src/index.ts

```
@@ -0,0 +1,723 @@
+        // Quick check for strings that don't need wrapping
+        if (stringWidth(string_) <= width) {
+            return [string_];
+        }
+
+        // First split by newlines to preserve them
+        const lines = string_.split(/\r?\n/);
+        const result: string[] = [];
+
+        // Precompile regex patterns
+        // eslint-disable-next-line no-control-regex,regexp/no-control-character
+        const colorPattern = /\u001B\[\d+m/g;
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)